d2renderers/d2svg/d2svg.go

```diff
@@ -991,14 +995,14 @@ func Render(diagram *d2target.Diagram) ([]byte, error) {
 	markers := map[string]struct{}{}
 	for _, obj := range allObjects {
 		if c, is := obj.(d2target.Connection); is {
-			labelMask := drawConnection(buf, c, markers, idToShape)
-			if labelMask != "" {
+			if labelMask := drawConnection(buf, c, markers, idToShape); labelMask != "" {
```

Comment:
i don't actually prefer these even if u can do it. it makes the line too long and it's harder to read. can u change these back?